tests/krishna_test.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+import sys
+
+sys.path.insert(1, '..//course-project-group-84//src')
+
+import game
+
+@pytest.fixture
+def test_init():
+    game.initialize()
+
+def test_with_fixture():
```

Comment:
I think a more descriptive test name could be helpful for organization purposes as we go on :D